settings.gradle

```diff
@@ -12,5 +12,5 @@ def includeModule(moduleName) {
 }
 
 includeModule("patient-management")
+includeModule("medical-records")
```

Comment:
Create documentation for how to introduce a new module

Reply:
Describe review process with minimal friction
- Keep PR small
- Discuss solution approach on GitHub ticket
- Explicit expectations: TDD
- Alternative design approaches are encouraged, but: within one module it should be a consistent code style, if there is a new pattern applied, e.g. persistence without ORM, it needs to be described in the arc42 documentation.
- Separation of API and implementation: e.g. Hibernate and Spring should not leak out of the implementation module.